frontend/src/views/pages/PaymentPage.tsx

```diff
@@ -418,17 +471,10 @@ const StyledOutlinedInput = styled(OutlinedInput)`
 `;
 
 const StyledOutlinedNumberInput = styled(StyledOutlinedInput)`
-  && input::-webkit-outer-spin-button,
-  && input::-webkit-inner-spin-button {
```

Comment:
You thought it is better to display arrows in an amount input? I just want to make sure and It's up to you.

// before
<img width="344" alt="Screenshot 2023-06-24 at 8 40 39 PM" src="https://github.com/e-0922-ja/expense-tracker-app/assets/112593974/1480813d-aedb-4df1-87a6-f49bb344ea63">

// after
<img width="330" alt="Screenshot 2023-06-24 at 8 32 25 PM" src="https://github.com/e-0922-ja/expense-tracker-app/assets/112593974/31102238-47e7-4c80-854e-c9067b76c05a">

Reply:
Sorry. Removed arrows.